cocos/2d/CCTMXObjectGroup.h

```diff
@@ -147,67 +159,71 @@ class CC_DLL TMXObjectGroup : public Node
      * @lua NA
      */
     virtual ~TMXObjectGroup();
-
-    /** Get the group name.
+    
+    /** Get the group name. 
      *
      * @return The group name.
      */
     inline const std::string& getGroupName() const { return _groupName; }
-
-    /** Set the group name.
+    
+    /** Set the group name. 
      *
      * @param groupName A string,it is used to set the group name.
      */
     inline void setGroupName(const std::string& groupName){ _groupName = groupName; }
 
-    /** Return the value for the specific property name.
+    /** Return the value for the specific property name. 
      *
      * @param propertyName The specific property name.
      * @return Return the value for the specific property name.
      * @js NA
      */
     Value getProperty(const std::string& propertyName) const;
+    
+    CC_DEPRECATED_ATTRIBUTE Value propertyNamed(const std::string& propertyName) const { return getProperty(propertyName); };
 
     /** Return the dictionary for the specific object name.
      * It will return the 1st object found on the array for the given name.
      *
      * @return Return the dictionary for the specific object name.
      */
     TMXObject* getObject(const std::string& objectName);
-
-    /** Gets the offset position of child objects.
+    
+    CC_DEPRECATED_ATTRIBUTE TMXObject* objectNamed(const std::string& objectName) { return getObject(objectName); };
+    
+    /** Gets the offset position of child objects. 
      *
      * @return The offset position of child objects.
      */
     inline const Vec2& getPositionOffset() const { return _positionOffset; };
-
-    /** Sets the offset position of child objects.
+    
+    /** Sets the offset position of child objects. 
      *
      * @param offset The offset position of child objects.
      */
     inline void setPositionOffset(const Vec2& offset) { _positionOffset = offset; };
-
-    /** Gets the list of properties stored in a dictionary.
+    
+    /** Gets the list of properties stored in a dictionary. 
      *
      * @return The list of properties stored in a dictionary.
      */
     inline const ValueMap& getProperties() const { return _properties; };
     inline ValueMap& getProperties() { return _properties; };
-
+    
     /** Sets the list of properties.
      *
      * @param properties The list of properties.
      */
     inline void setProperties(const ValueMap& properties) {
         _properties = properties;
     };
-
-    /** Gets the array of the objects.
+    
+    /** Gets the array of the objects. 
      *
      * @return The array of the objects.
      */
-    Vector<TMXObject*> getObjects();
-    
+    inline const Vector<TMXObject*> getObjects() { return _objects; };
```

Comment:
Is there any reason that you don't return a reference or const reference? e.g.
```c++
const Vector<TMXObject*>&
Vector<TMXObject*>&
```

Otherwise, the return vector will be a new vector and will be slower.